src/cli.ts

```diff
@@ -3,12 +3,50 @@
 import chalk from "chalk";
 import program from "commander";
 import fs from "fs-extra";
+import { RewriteRule, RewriteRules } from "./rewrite";
 import { RecordReplayServer } from "./server";
 
 function commaSeparatedList(value: string) {
   return value ? value.split(",") : [];
 }
 
+const RE_REWRITE_RULE = /s\/(.+(?<!\\))\/(.+(?<!\\))\/([gims]*)/;
+const RE_REPLACE_RULE = /(\\[1-9][0-9]*)/;
+
+function rewriteRule(value: string, rewriteRules: RewriteRules): RewriteRules {
+  // Does the given value match a sed-style regex expression?
+  const match = RE_REWRITE_RULE.exec(value);
+  if (match === null) {
+    throw new Error(
+      `Provided rewrite rule ${value} does not look like a sed-like rewrite rule.`
+    );
+  }
+  const [rawFind, rawReplace, rawFlags] = match.slice(1, 4);
```

Comment:
**Question** How do you know this is 1 and 4?

Reply:
This is part of the JavaScript regex API. Regex capture groups are numbered from 1 onwards. Regex capture group 0 is the full string. `RE_SED_INPUT_VALIDATION` contains three capture groups. `s/<one>/<two>/<three>`.